### ISSUE TITLE
Please provide debian packages

### ISSUE THREAD

**Issue by Polve** [backuppc/backuppc#184]
To avoid to self compile the program and simplify its installation and hence its usage, would it be possible to have updated debian packages?

Even better would be, if possible, to collaborate with debian package managers so to have it in the standard distribution.

**Comment by raoulbhatia** [comment]
Hi @Polve .

First steps have been done to get libbackuppc-xs-perl into Debian, see https://packages.debian.org/search?keywords=backuppc

For the remainder, could you try following https://github.com/backuppc/backuppc/wiki/Build-Your-Own-Packages and let me know if this still works for Debian?  I am running Ubuntu and cannot verify right now.

Thanks,
Raoul

**Comment by Jim-McNamara** [comment]
Hello, folks. 

Raoul, I followed the guide to building your own packages on a default Debian machine. It mostly worked. Out of the gate, I had to install git. Both rsync-bpc and backuppc-xs installed fine. When I tried installing the main backuppc deb I received a message that I was missing libtime-modules-perl and rrdtool. I told apt to install those 2 programs. The rrdtool install broke apt over a conflicting version (sorry disappeared from the screen in the VM). Issuing the command:

apt --fix-broken install

cleared that up, then the backuppc .deb successfully installed. After restarting apache2 I was able to see the main page.
